README.md

```diff
@@ -85,25 +85,30 @@ add them to the NEXTCLOUD_AUTOINSTALL_APPS variable in `.env`.
 
 ## Complex setup
 
-In order to achieve a more complex dev environment with different branches of the server, follow the manual steps:
+In order to achieve a more complex dev environment with different branches of the server, follow the following manual steps:
 
-##### 1. Clone the repository
+##### 1. Clone and prepare the repository
+
+1. Run the setup from the "Simple master setup" from above.
+2. For this setup it is nessesary to do the steps that are descripte in the note from the "Simple master setup".
 
 ```
-git clone https://github.com/nextcloud/server.git
+cd workspace/server
+git fetch --unshallow
+git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
+git fetch origin
+pwd
+cd ../..
 ```
+Please note the printing of the pwd command for step 3 as `REPO_PATH_SERVER`.
```

Comment:
`REPO_PATH_SERVER` should be properly set already by the bootstrap.sh script, so I think this can be omitted.